.changeset/breezy-chairs-eat.md

```diff
@@ -0,0 +1,5 @@
+---
+"@chromatic-com/shared-e2e": patch
```

Comment:
Maybe it wasn't clear in the docs, but the shared package doesn't get published, only the cypress and playwright packages do. So when shared changes you also need to mark the others as changed so they are published.

Reply:
Ahh yeah it wasn't clear - the docs just said to follow along with the changeset dialog and wasn't sure. I was thinking the shared package got updated and the others used the new update. I'll make a change here!